src/main/java/com/getindata/flink/connector/jdbc/catalog/IndexFilterResolver.java

```diff
@@ -0,0 +1,52 @@
+package com.getindata.flink.connector.jdbc.catalog;
+
+import org.apache.flink.shaded.guava30.com.google.common.base.Strings;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static java.util.Collections.emptyList;
+
+class IndexFilterResolver {
+
```

Comment:
Nitpick I quess I would place 2 private fields at top? :thinking:

Reply:
Nope. Standard order of java class members is as follows: https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html section: 3.1.3